frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -36,8 +36,7 @@ import scala.collection.concurrent.TrieMap
 import scala.collection.JavaConverters._
 import scala.concurrent.Promise
 import scala.concurrent.duration.FiniteDuration
-import scala.util.Success
-import scala.util.Failure
+import scala.util.{Try, Success, Failure}
```

Comment:
Let's not change imports - I think single import on line is the standard in the repo.

Reply:
It usually is the standard because almost all imports come from Metals, aggregating all of them in one import doesn't bother me, though I agree with the sentiment :smile: For the record, it is customary to avoid wildcard imports in the codebase, leaning always towards explicit imports.